TLC59116.h

```diff
@@ -403,7 +402,8 @@ class TLC59116 : public TLC59116_Unmanaged {
     */
     TLC59116& set_milliamps(byte ma, int Rext=Rext_Min); // Rext_Min ohms is 120ma at reset.
 
-    int milliamps(int Rext=Rext_Min) {return i_out(shadow_registers[IREF_Register]); } ///< The calculated current setting
+	// MUST call resync_shadow_registers or this will return stale data
+    int milliamps(int Rext=Rext_Min) {resync_shadow_registers(); return i_out(shadow_registers[IREF_Register], Rext); } ///< The calculated current setting
```

Comment:
(note to self: check if this semantically makes sense, resync is expensive)

Reply:
May be done better elsewhere...but without resync it was not returning.  Also note that the original version wasn't passing in Rext